relay-general/src/pii/processor.rs

```diff
@@ -100,13 +100,13 @@ impl<'a> Processor for PiiProcessor<'a> {
         self.apply_all_rules(meta, state, Some(value))
     }
 
-    fn process_native_image_path(
```

Comment:
I think I regret this rename actually, can we go with nativeimagepath or imagepath all the way? The reason being that there's special behavior attached to this valuetype that makes no sense in other situations outside of filepaths that are relevant for processing.

@jan-auer wdyt? Intent is to let people type `$native_image_path` or something like that. Right now they write `debug_meta.images.*.code_file` which I think we should not emulate for minidump scrubbing

Reply:
I would definitely not expose this value type to data scrubbers, this should be fully embedded in processing. Otherwise, it breaks left and right if we have to evolve this later. 

What we **could** do is introduce a `$path` type, but then everything that could contain a path would probably have to be tagged by that. And I think we have attributes like `$frame.package` that can be either or, where it not necessarily makes sense to apply this logic. So I'd stay on the safer side and not do it. 

Particularly, the `$frame.package` attribute can contain a full absolute native image path. We cannot classify it as such, though, since it can also contain arbitrary other information in other platforms. 

Really, this is a matcher. Not a selector, and not a value type. We should have a rule where you can say "In [selector] if this matches a file location, then on the path apply [redaction]".

Reply:
Ok you bring up some good points. I think I know what I want. I will push a solution and summarize later.

Reply:
So to summarize:

* nativeimagepath stays as-is
* no valuetype, that means people will need to explicitly add `$attachments.*.code_file` (or just `code_file`)